plugins/k8sauditeks/README.md

```diff
@@ -0,0 +1,79 @@
+# Kubernetes Audit Events Plugin for EKS
+
+## Introduction
+
+This plugin extends Falco to support [Kubernetes Audit Events](https://kubernetes.io/docs/tasks/debug-application-cluster/audit/#audit-backends) from AWS EKS clusters as a new data source. 
+For more details about what Audit logs are, see the [README of k8saudit plugin](https://github.com/falcosecurity/plugins/blob/master/plugins/k8saudit/README.md).
+
+### Functionality
+
+This plugin supports consuming Kubernetes Audit Events stored in Cloudwatch Logs for the EKS Clusters, see [AWS official documentation](https://docs.aws.amazon.com/eks/latest/userguide/control-plane-logs.html) for details.
+
+## Capabilities
+
+The `k8sauditeks` uses the field extraction methods of the [`k8saudit`](https://github.com/falcosecurity/plugins/tree/master/plugins/k8saudit) plugin as the format for the Audit Logs is same.
+
+### Event Source
+
+The event source for Kubernetes Audit Events from EKS is `k8s_audit`, it allows to use same rule than `k8saudit` plugin.
+
+### Supported Fields
+
+See the list [here](https://github.com/falcosecurity/plugins/tree/master/plugins/k8saudit#supported-fields).
+
+## Usage
+
+### Configuration
+
+Here's an example of configuration of `falco.yaml`:
+
+```yaml
+plugins:
+  - name: k8sauditeks
+    library_path: libk8sauditeks.so
+    init_config:
+      region: "us-east-1"
+      profile: "default"
+      cluster_name: "my-cluster"
+      shift: 10
+      polling_interval: 10
+      use_async: false
+      buffer_size: 500
+    open_params: ""
+  - name: json
+    library_path: libjson.so
+    init_config: ""
+
+load_plugins: [k8sauditeks, json]
+```
+
+**Initialization Config**:
+ * `region`: The Region of your EKS cluster (default: us-east-1)
+ * `profile`: The AWS Profile to use
+ * `cluster_name`: The name of your EKS Cluster
+ * `use_async`: If true then async extraction optimization is enabled (Default: true)
+ * `polling_interval`: Polling Interval in seconds (default: 5s)
+ * `shift`: Time shift in past in seconds (default: 1s)
+ * `buffer_size`: Buffer Size (default: 200)
+
+### Rules
+
+The `k8sauditeks` plugin ships with a default set of ruleset (see `rules/` directory). These rules are same than those for `k8saudit`, only the required plugin changes.
```

Comment:
Replicating the whole ruleset is a bad choice here. This plugin should not ship any ruleset, and be compatible with the `k8saudit` one. I see why you couldn't do it, and I opened an issue for this problem specifically: https://github.com/falcosecurity/falco/issues/2150

So I think our choice here would be to:
- Block this PR until https://github.com/falcosecurity/falco/issues/2150 is merged
- Remove the `k8saudit` plugin dependency on the ruleset and keep only `json`, so that this plugin gets "implicitly" accepted too

Reply:
I'm off for next 2 weeks, we can wait til my return

Reply:
I replaced the ruleset by a single dummy rule to allow people to test the plugin, we could change that in future, wdyt?

Reply:
I think the way to go for now would be to remove the `k8s_audit` requirement from `required_plugin_versions` of the actual k8saudit ruleset, so that users can test this against the real k8s audit rules and we don't have an unused dummy ruleset here.

Reply:
I agree, but what can we do until that? I don't want to block this PR too long

Reply:
You should be all set with the new feature we introduced in Falco with https://github.com/falcosecurity/falco/pull/2190. It will be part of the upcoming Falco 0.33.0 release.

In your case, you should attain what you want by editing the official k8saudit ruleset with something like:
```yaml
- required_plugin_versions:
  - name: k8saudit
    version: 0.1.0
    alternatives:
      - name: k8saudit-eks
        version: 0.1.0
```

Reply:
> You should be all set with the new feature we introduced in Falco with [falcosecurity/falco#2190](https://github.com/falcosecurity/falco/pull/2190). It will be part of the upcoming Falco 0.33.0 release.

I agree, it should work as is, I tried to respect

> In your case, you should attain what you want by editing the official k8saudit ruleset with something like:
> 
> ```yaml
> - required_engine_version: 15
> - required_plugin_versions:
>   - name: k8saudit
>     version: 0.1.0
>     alternatives:
>       - name: k8saudit-eks
>         version: 0.1.0
> ```

Ok, let me update the README to explain that